helix-term/src/main.rs

```diff
@@ -114,24 +116,40 @@ FLAGS:
 
     setup_logging(logpath, args.verbosity).context("failed to initialize logging")?;
 
-    let config_dir = helix_loader::config_dir();
-    if !config_dir.exists() {
-        std::fs::create_dir_all(&config_dir).ok();
-    }
-
-    let config = match std::fs::read_to_string(config_dir.join("config.toml")) {
-        Ok(config) => toml::from_str(&config)
-            .map(helix_term::keymap::merge_keys)
-            .unwrap_or_else(|err| {
-                eprintln!("Bad config: {}", err);
-                eprintln!("Press <ENTER> to continue with default config");
-                use std::io::Read;
-                // This waits for an enter press.
-                let _ = std::io::stdin().read(&mut []);
-                Config::default()
-            }),
-        Err(err) if err.kind() == std::io::ErrorKind::NotFound => Config::default(),
-        Err(err) => return Err(Error::new(err)),
+    let config_value = local_config_dirs()
+        .into_iter()
+        .chain([config_dir()].into_iter())
+        .map(|path| path.join("config.toml"))
+        .rev()
```

Comment:
Why call `rev()`? I think that would make the local configs be overridden by the upper configs.